sdk/python/symbolchain/nem/ExtendedTypeParsingRules.py

```diff
@@ -0,0 +1,56 @@
+import importlib
+import inspect
+
+from ..BaseValue import BaseValue
+from ..CryptoTypes import Hash256, PublicKey
+from ..TypeParserBuilder import TypeParserBuilder
+from .Network import Address
+
+
+# TODO: probably also should be extracted to TypeParserBuilder
+def _wrap_sdk_type(type_name, base_type_parsing_rules):
+	if base_type_parsing_rules and type_name in base_type_parsing_rules:
+		return base_type_parsing_rules[type_name]
```

Comment:
why is this needed for nem but not symbol?

Reply:
it's needed for both, changed already, mentioned it earlier in one of TODOs